src/templates/city.js

```diff
@@ -2,16 +2,17 @@ import React from 'react'
 import { graphql } from 'gatsby'
 import Main from '../pages/_main'
 
-export default ({ data = {} }) => {
+// the redirection to event wasn;t working so passed the query
```

Comment:
I think that if passing the query resolves the redirect issues that this comment can be removed.